app/v1/models/hateoas/RelType.scala

```diff
@@ -18,6 +18,7 @@ package v1.models.hateoas
 
 object RelType {
   val SAMPLE_ENDPOINT_REL = "sample-rel"
-
   val SELF = "self"
+  val AMEND_OTHER_DEDUCTIONS = "amend-other-deductions"
+  val DELETE_OTHER_DEDUCTIONS = "amend-other-deductions"
```

Comment:
The rel in the hateoas glossary example has it as deductions-other not other-deductions